js/app.js

```diff
@@ -4,3 +4,78 @@
 console.log('App Started');
 console.log('Fruit count', fruits.length);
```

Comment:
Here is how I moved from one array to the next (to the fruit column):

    $scope.toFruit = function(idx){
        $scope.allFruit.push($scope.fruitAndVeg[idx]);
        $scope.fruitAndVeg.splice(idx, 1);
        if ($scope.fruitAndVeg.length == 0) {
            $scope.winCheck();
        }
    }